tensorflow/go/attrs.go

```diff
@@ -0,0 +1,205 @@
+			storage := make([]C.char, meta.total_size)
+			C.TF_OperationGetAttrStringList(op.c, cname, &values[0], &lengths[0], C.int(meta.list_size), unsafe.Pointer(&storage[0]), C.size_t(meta.total_size), status.c)
+			if err := status.Err(); err != nil {
+				return nil, err
+			}
+			list := make([]string, meta.list_size)
+			for i, val := range values {
+				length := lengths[i]
+				list[i] = C.GoStringN((*C.char)(val), C.int(length))
+			}
+			return list, nil
+
+		case C.TF_ATTR_INT:
+			list := make([]C.int64_t, meta.list_size)
+			C.TF_OperationGetAttrIntList(op.c, cname, &list[0], C.int(meta.list_size), status.c)
+			if err := status.Err(); err != nil {
+				return nil, err
+			}
+			vals := make([]int64, meta.list_size)
+			for i, val := range list {
+				vals[i] = int64(val)
+			}
+			return vals, nil
+
+		case C.TF_ATTR_FLOAT:
+			list := make([]C.float, meta.list_size)
+			C.TF_OperationGetAttrFloatList(op.c, cname, &list[0], C.int(meta.list_size), status.c)
+			if err := status.Err(); err != nil {
+				return nil, err
+			}
+			vals := make([]float32, meta.list_size)
+			for i, val := range list {
+				vals[i] = float32(val)
+			}
+			return vals, nil
+
+		case C.TF_ATTR_BOOL:
+			list := make([]C.uchar, meta.list_size)
+			C.TF_OperationGetAttrBoolList(op.c, cname, &list[0], C.int(meta.list_size), status.c)
+			if err := status.Err(); err != nil {
+				return nil, err
+			}
+			vals := make([]bool, meta.list_size)
+			for i, val := range list {
+				vals[i] = val == 1
+			}
+			return vals, nil
+
+		case C.TF_ATTR_TYPE:
+			list := make([]C.TF_DataType, meta.list_size)
+			C.TF_OperationGetAttrTypeList(op.c, cname, &list[0], C.int(meta.list_size), status.c)
+			if err := status.Err(); err != nil {
+				return nil, err
+			}
+			vals := make([]DataType, meta.list_size)
+			for i, val := range list {
+				vals[i] = DataType(val)
+			}
+			return vals, nil
+
+		case C.TF_ATTR_TENSOR:
+			list := make([]*C.TF_Tensor, meta.list_size)
+			C.TF_OperationGetAttrTensorList(op.c, cname, &list[0], C.int(meta.list_size), status.c)
+			if err := status.Err(); err != nil {
+				return nil, err
+			}
+			vals := make([]*Tensor, meta.list_size)
+			for i, t := range list {
+				vals[i] = newTensorFromC(t)
+			}
+			return vals, nil
+
+		case C.TF_ATTR_SHAPE:
+			dims := make([]*C.int64_t, meta.list_size)
+			numDims := make([]C.int, meta.list_size)
+			storage := make([]C.int64_t, meta.total_size)
+			C.TF_OperationGetAttrShapeList(op.c, cname, &dims[0], &numDims[0], C.int(meta.list_size), &storage[0], C.int(meta.total_size), status.c)
+			if err := status.Err(); err != nil {
+				return nil, err
+			}
+			list := make([]Shape, meta.list_size)
+			for i, dim := range dims {
+				numDim := numDims[i]
+				// A []C.int64_t slice backed by C memory.
+				// See: https://github.com/golang/go/wiki/cgo#turning-c-arrays-into-go-slices
+				slice := (*[1 << 30]C.int64_t)(unsafe.Pointer(dim))[:numDim:numDim]
+				s := Shape{
```

Comment:
Nit: Fit in one line (or maybe just run `gofmt` on this file?)

Reply:
It was already go fmted, but done. Though, I think it's cleaner with it spaced out.